### ISSUE TITLE
get color palette 

### ISSUE THREAD

**Issue by dridk** [ColinDuquesnoy/QDarkStyleSheet#171]
Information about your current environment setup:
  - QDarkStyle: 2.6.8
  - OS: Linux 4.15.0-50-generic x86_64
  - Platform: linux
  - Python: 3.6.7.final.0
  - Python API: 1013
  - Binding in use:     Not set or nonexistent
  - Abstraction in use: Not set or nonexistent
  - qtpy (QT_API):                Not set or nonexistent
  - pyqtgraph (PYQTGRAPH_QT_LIB): Not set or nonexistent
  - Qt.py (__binding__):          Not set or nonexistent

Qt bindings available:
  - PySide2

### Description / Steps to Reproduce 

I cannot access palette color of the current style. For example, if I try to access the text color supposed to be light, I get only the dark color from standard light theme. 

```python
app = QApplication(sys.argv)
app.setStyleSheet(qdarkstyle.load_stylesheet_pyside2())

# This line should return the light text color of darktheme style.  But it doesn't .
# It returns only the default light theme palette
print(app.palette().color(QPalette.Text).name()) 


```

**Comment by goanpeca** [comment]
@dridk I do not think that you can get those values from the qpalette. If you need to access the palette programmatically you can use 

https://github.com/ColinDuquesnoy/QDarkStyleSheet/blob/master/qdarkstyle/palette.py

```python
from qdarkstyle.palette import DarkPalette
```

**Comment by dridk** [comment]
Hi, Thanks for your reply . 
I just swtiched to qtmodern . I think you must follow the same pattern. 
They first set a QPalette from Python and secondly access colors from qss file using the "palette" keyword . 

```
QGroupBox {
  background-color: palette(alternate-base);
  border: 1px solid palette(midlight);
  margin-top: 25px;
}
```
https://github.com/gmarull/qtmodern/blob/master/qtmodern/resources/style.qss

So now, I can be agnostic from the style and get color from QPalette.

**Comment by goanpeca** [comment]
Thanks for pointing to th-at . resource. Thoughts @dpizetta

**Comment by dpizetta** [comment]
I'll take a look. Tks
